airbyte-integrations/connectors/source-opsgenie/setup.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from setuptools import find_packages, setup
+
+MAIN_REQUIREMENTS = [
+    "airbyte-cdk~=0.1.56",
+]
+
+TEST_REQUIREMENTS = [
+    "pytest~=6.1",
+    "pytest-mock~=3.6.1",
+    "source-acceptance-test",
+    "responses~=0.19.0",
+]
+
+setup(
+    name="source_opsgenie",
+    description="Source implementation for Opsgenie.",
+    author="Airbyte",
+    author_email="contact@airbyte.io",
+    packages=find_packages(),
+    install_requires=MAIN_REQUIREMENTS,
+    package_data={"": ["*.json", "*.yaml", "schemas/*.json", "schemas/shared/*.json"]},
+    extras_require={
+        "tests": TEST_REQUIREMENTS,
+    },
+)
```

Comment:
Please reinstate:

```py


dependencies {
    implementation files(project(':airbyte-integrations:bases:source-acceptance-test').airbyteDocker.outputs)
}
```

Reply:
Is this a copypasta? I'm not sure where this Gradle snippet needs to go?

Reply:
@adamschmidt look at other connectors gradle files, i don’t think your gradle needs to be modified. Will go at the end after all other condition's.